src/components/BrowserFilter/BrowserFilter.react.js

```diff
@@ -118,7 +138,11 @@ export default class BrowserFilter extends React.Component {
                 renderRow={props => (
                   <FilterRow {...props} active={this.props.filters.size > 0} parentContentId={POPOVER_CONTENT_ID} />
                 )}
-              />
+                />
+                {this.state.confirmName && <Field
+                  label={<Label text='What should we this filter?' />}
```

Comment:
Let's give these "views" a distinct name, this will help to linguistically distinguish between the filter dialog and the new filter view on the side. If "View" is too general, maybe "Filter view"?

Also, I find many dialogs and buttons to be too verbose in Parse Dashboard, so maybe let's try to make them more concise, for example:

```suggestion
                  label={<Label text='Filter view name:' />}
```